packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -82,6 +108,9 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
         () {
       final newFrame = _controller.frameBasedTimeline.frameAddedNotifier.value;
       if (newFrame == null) return;
+
+      _setupFPSHighwaterLine();
+
```

Comment:
I don't think this should be computed in the frame callback. Seems like we should compute in didChangeDependencies. Later once offline is supported, this will need to be recomputed on clear

Reply:
Yes I think that's better too.